packages/react-component-library/src/components/Toast/partials/StyledToast.tsx

```diff
@@ -0,0 +1,58 @@
+import styled, { css } from 'styled-components'
+import { selectors } from '@royalnavy/design-tokens'
+import { AppearanceTypes } from 'react-toast-notifications'
+
+import { StyledToastLabel } from './StyledToastLabel'
+
+interface StyledToastProps {
+  $appearance?: AppearanceTypes
+}
+
+const { shadow, color, spacing } = selectors
+
+export const StyledToast = styled.div<StyledToastProps>`
+  box-shadow: ${shadow('2')};
+  display: flex;
+  flex-direction: column;
+  border: 1px solid ${color('neutral', '100')};
+  border-radius: 4px;
+  width: 340px;
+  margin-bottom: ${spacing('6')};
+  background-color: ${color('neutral', 'white')};
+
+  ${({ $appearance }) => {
+    if ($appearance === 'success') {
```

Comment:
I wonder if these `if` statements could be consolidated with some kind of lookup because all that is changing is the first parameter to the `colour` `function`.